_examples/arrays/test.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2017 The go-python Authors.  All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+from __future__ import print_function
+import arrays
+
+a = [1,2,3,4]
```

Comment:
I would put the same content in the python list and in the Go array (will be less confusing if/when the test fails later on)